views/dist/back.css.map

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"version":3,"file":"back.css","mappings":"AAkBA,0CAEG,aAAc,CACd,UAAW,CACX,aAAc,CACd,iBAAkB,CAClB,sBAAuB,CACvB,yEAAiF,CACjF,iEAAyE,CACzE,yBAA0B,CAC1B,sBAAuB,CACvB,qBAAsB,CACtB,oBAAqB,CACrB,iBAAkB,CACjB,cAAe,CAdnB,yBAkBG,eAAgB,CAChB,uBAAwB,CACxB,kBAAmB,CACnB,wBAAyB,CACzB,wBAAyB,CACzB,iBAAkB,CAClB,mDAAuD,CACvD,2CAA+C,CAzBlD,gCA4BK,iBAAkB,CA5BvB,mCAkCM,YAAa,CACd,gBAIJ,YAAa","sources":["webpack://ps_facetedsearch/./_dev/back/blocklayered.scss"],"sourcesContent":["/**\n * Copyright since 2007 PrestaShop SA and Contributors\n * PrestaShop is an International Registered Trademark & Property of PrestaShop SA\n *\n * NOTICE OF LICENSE\n *\n * This source file is subject to the Academic Free License 3.0 (AFL-3.0)\n * that is bundled with this package in the file LICENSE.md.\n * It is also available through the world-wide-web at this URL:\n * https://opensource.org/licenses/AFL-3.0\n * If you did not receive a copy of the license and are unable to\n * obtain it through the world-wide-web, please send an email\n * to license@prestashop.com so we can send you a copy immediately.\n *\n * @author    PrestaShop SA <contact@prestashop.com>\n * @copyright Since 2007 PrestaShop SA and Contributors\n * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)\n */\n.bootstrap {\n  .filter_list .filter_list_item {\n\t  display: table;\n\t  width: 100%;\n\t  padding: 5px 0;\n\t  margin-bottom: 4px;\n\t  background-color: white;\n\t  -webkit-box-shadow: rgba(0, 0, 0, 0.3) 0 0 3px, rgba(0, 0, 0, 0.1) 0 -2px 0 inset;\n\t  box-shadow: rgba(0, 0, 0, 0.3) 0 0 3px, rgba(0, 0, 0, 0.1) 0 -2px 0 inset;\n\t  -webkit-border-radius: 3px;\n\t  -moz-border-radius: 3px;\n\t  -ms-border-radius: 3px;\n\t  -o-border-radius: 3px;\n\t  border-radius: 3px;\n    cursor: pointer;\n  }\n\n  .filter_panel {\n\t  min-height: 20px;\n\t  padding: 7px 7px 0px 7px;\n\t  margin-bottom: 20px;\n\t  background-color: #ebebeb;\n\t  border: 1px solid #d9d9d9;\n\t  border-radius: 3px;\n\t  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);\n\t  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);\n\n    header {\n\t    margin-bottom: 7px;\n    }\n  }\n\n  .prestashop-switch {\n    span {\n      display: none;\n    }\n  }\n}\n.sortable-ghost {\n\tcolor: orange;\n}\n"],"names":[],"sourceRoot":""}
```

Comment:
Do we need to version this file?

Reply:
It is unneeded if `npm run build` is run automatically in workflows.